config/chime_science_run_gpu.yaml

```diff
@@ -410,6 +410,14 @@ gpu_rfi_output_buffers:
     kotekan_buffer: standard
   gpu_rfi_output_buffer_3:
     kotekan_buffer: standard
+  rfi_value_buffer_0:
+    kotekan_buffer: standard
+  rfi_value_buffer_1:
+    kotekan_buffer: standard
+  rfi_value_buffer_2:
+    kotekan_buffer: standard
+  rfi_value_buffer_3:
+    kotekan_buffer: standard
```

Comment:
I would set the `num_frames` value for these to be something much smaller.  Maybe just: `num_frames: buffer_depth`